src/meshgrids/imfreq.jl

```diff
@@ -64,7 +64,7 @@ function ImFreq(β, isFermi::Bool=false;
         error("Proper grid or basis are required.")
     end
 
-    @assert issorted(grid) "The grid should be sorted."
+    @assert issorted(grid) || issorted(grid, rev=true) "The grid should be sorted."
```

Comment:
Reverse ordered grids not yet supported in CompositeGrids.
This may cause problem when floor and interp are envolved.

Reply:
It should be okay. User will see an error message if they need floor and interp. For many applications, like SYK solver, there is no need to use floor and interp

Reply:
The problem is they will not. Currently CompositeGrids.SimpleG.Arbitrary do not complain(which should be improved) when it gets an unsorted array, nor do floor and interp. Thus it's dangerous to leave it at current status, as the program could continue running and produce some garbage or inaccurate results with no warning.
We should either add support in CompositeGrids, or in GreenFunc to support reversed grid.